tests/apollo/util/concord_msgs.py

```diff
@@ -758,6 +758,97 @@ def __eq__(self, other):
             return False
         return True
 
+class InstallCommand():
```

Comment:
Nit: `UpgradeCommand` class and its usage should be removed in `concord_msgs.py`, since we are using `InstallCommand`.

Reply:
done